### PR TITLE
Fix/collapsible alignment

### PR DESCRIPTION
**Pull request by talevy17**

### CODE REVIEW

File: packages/plugin-collapsible-list/web/src/tiptap/defaults.ts [hunk 1]
```diff
@@ -16,6 +16,11 @@ export const defaultCollapsibleItem = [
           {
             type: Node_Type.PARAGRAPH,
             content: [],
+            attrs: {
```
Comment (danarish4) [inline packages/plugin-collapsible-list/web/src/tiptap/defaults.ts:19]: remove

File: packages/plugin-collapsible-list/web/src/tiptap/component.tsx [hunk 2]
```diff
@@ -49,15 +49,20 @@ export const CollapsibleList: React.FC<PluginProps> = ({
 
   const direction = componentData.config?.direction || languageDir;
 
+  useEffect(() => {
+    editor.commands.normalizeCollapsibleListNodeAlignment(node);
+  }, []);
+
   return (
     <div
+      dir={direction}
```
Comment (danarish4) [inline packages/plugin-collapsible-list/web/src/tiptap/component.tsx:58]: check if you can cleanup

File: packages/plugin-collapsible-list/web/src/tiptap/defaults.ts (hunk 1 above)
Comment (talevy17) [reply packages/plugin-collapsible-list/web/src/tiptap/defaults.ts:20]: kept it and wired it to languageDir for consistency

File: packages/plugin-collapsible-list/web/src/tiptap/component.tsx (hunk 2 above)
Comment (talevy17) [reply packages/plugin-collapsible-list/web/src/tiptap/component.tsx]: removed

**Pull request merged**